samples/msgext-ai-sentiment-analysis/nodejs/.vscode/tasks.json

```diff
@@ -0,0 +1,105 @@
+// This file is automatically generated by Teams Toolkit.
```

Comment:
Please check if we need to push this file?

Reply:
This file has Generated by default and, it is in your PR as well.

Reply:
ok